src/components/SummaryPanel/InOutRateChart.tsx

```diff
@@ -1,88 +1,88 @@
 import * as React from 'react';
-import { StackedBarChart } from 'patternfly-react';
+import { Chart, ChartBar, ChartStack, ChartAxis, ChartTooltip } from '@patternfly/react-charts';
 import { PfColors } from '../../components/Pf/PfColors';
-import { LegendPosition, SUMMARY_PANEL_CHART_WIDTH } from '../../types/Graph';
+import { SUMMARY_PANEL_CHART_WIDTH } from '../../types/Graph';
+import * as Legend from 'components/Charts/LegendHelper';
 
 type InOutRateChartGrpcPropType = {
-  height?: number;
```

Comment:
@israel-hdez @jshaughn For simplicity I've removed these properties as they were not used (so never tested), but if you don't like that and prefer that I take them back, I will.